service/frontend/workflow_handler.go

```diff
@@ -2758,6 +2758,10 @@ func (wh *WorkflowHandler) DescribeSchedule(ctx context.Context, request *workfl
 		return nil, err
 	}
 
+	err = wh.annotateSearchAttributesOfScheduledWorkflow(&queryResponse, request.GetNamespace())
+	if err != nil {
+		return nil, fmt.Errorf("describe schedule:%w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("describe schedule: %w", err)
```

I think the error would end up being more readable if there is a space here.

For example: `describe schedule:annotate search attributes:an error` vs `describe schedule: annotate search attributes: an error`.

(same for the ones below)